rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -59,15 +63,20 @@
 	 * @return {@link List} of unprocessed {@link Metadata}
 	 */
 	public List<Metadata> getUnprocessedCpcPlusMetaData() {
-		HashMap<String, AttributeValue> valueMap = new HashMap<>();
-		valueMap.put(":cpcValue", new AttributeValue().withN("1"));
-		valueMap.put(":cpcProcessedValue", new AttributeValue().withN("0"));
 
-		DynamoDBScanExpression metadataScan = new DynamoDBScanExpression()
-				.withFilterExpression("Cpc = :cpcValue and CpcProcessed = :cpcProcessedValue")
-				.withExpressionAttributeValues(valueMap);
+		return IntStream.range(0, Constants.CPC_DYNAMO_PARTITIONS).mapToObj(partition -> {
+			HashMap<String, AttributeValue> valueMap = new HashMap<>();
+			valueMap.put(":cpcValue", new AttributeValue().withS(Constants.CPC_DYNAMO_PARTITION_START + partition));
+			valueMap.put(":cpcProcessedValue", new AttributeValue().withS("false"));
+
+			DynamoDBQueryExpression<Metadata> metadataQuery = new DynamoDBQueryExpression<Metadata>()
```

Comment:
new DynamoDBQueryExpression<>()

Reply:
That actually doesn't work.  I get a compilation error.  I don't know why.